src/commcare_cloud/fab/fabfile.py

```diff
@@ -398,7 +398,7 @@ def setup_release(keep_days=0):
     _setup_release(parse_int_or_exit(keep_days), full_cluster=True)
 
 
-def _setup_release(keep_days=0, full_cluster=True):
+def _setup_release(keep_days=2, full_cluster=True):
```

Comment:
was this change intentional?

Reply:
Yup, this was an important part of the change. The idea was to make sure a celery process cannot outlive the code that underlies it. A bit more motivation in https://dimagi-dev.atlassian.net/browse/SAAS-11324 and the p1 ticket it links to.